tests/VObject/VCardBirthdayTest.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace Sabre\VObject;
+
+class VCardBirthdayTest extends TestCase {
+
+    function testValidBirthday() {
+
+        $input = <<<VCF
+BEGIN:VCARD
+VERSION:3.0
+N:Gump;Forrest;;Mr.
+FN:Forrest Gump
+BDAY:19850407
```

Comment:
Haha, no, I just added an easter egg and with your name it would have been too obvious. ;-)